samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -220,11 +232,13 @@ The user interface of the sample depends on the hardware platform you are using.
          * Blinks at a 0.25 second interval if the DFU mode is enabled.
 
       LED 2:
-         Indicates that the ringing action is in progress.
-         Depending on the ringing state:
+         Indicates that the ringing action is in progress, informs that the simulated motion has been detected, and informs that the motion detector is active.
+         Depending on the motion detected event, the ringing state, and the motion detector state:
 
-         * Lit if the device is ringing.
-         * Off if the device is not ringing.
+         * Blinks fast twice if the motion detected event appears.
+         * Lit if the device is ringing and motion detected event is not being signalized.
```

Comment:
"signalized" sounds a bit odd, but cannot figure out what would be better.

Reply:
Maybe:

```suggestion
         * Lit if the device is ringing and no motion detection event is being indicated.
```

? We could also stick to "signal", but then I think it should be "signaled", instead of "signalized"

Reply:
Indicated sounds better. Thanks!